docs/API.md

```diff
@@ -69,6 +69,14 @@ protocol stream.
     Value `0` implies low quality and `9` implies high quality.
     Default value is `6`.
 
+`compressionLevel`
+  - Is an `int` in range `[0-9]` controlling the desired compression
+    level. Value `0` means no compression. Level 1 uses a minimum of CPU
+    resources and achieves weak compression ratios, while level 9 offers
+    best compression but is slow in terms of CPU consumption on the server
+    side. Use high levels with very slow network connections. This only
+    applies to connections using the Tight encoding. Default value is `2`.
```

Comment:
The server can use the hint for whatever it likes, not just Tight. Otherwise LGTM. :+1:

Reply:
fixed